src/server/pkg/storage/fileset/option.go

```diff
@@ -21,6 +21,18 @@ func WithShardThreshold(threshold int64) StorageOption {
 	}
 }
 
+//func WithLevelZeroSize(size int64) StorageOption {
+//	return func(s *Storage) {
+//		s.levelZeroSize = size
+//	}
+//}
+//
+//func WitLevelSizeBase(base int) StorageOption {
+//	return func(s *Storage) {
+//		s.levelSizeBase = base
+//	}
+//}
+
```

Comment:
Can this be uncommented?

Reply:
I don't remember why I commented this. I usually make a note. I think it should be fine to expose these options.